saleor/graphql/page/tests/test_page.py

```diff
@@ -113,21 +113,68 @@ def test_customer_query_unpublished_page(user_api_client, page):
     assert content["data"]["page"] is None
 
 
-def test_staff_query_unpublished_page(staff_api_client, page):
+def test_staff_query_unpublished_page_by_id(
+    staff_api_client, page, permission_manage_pages
+):
+    page.is_published = False
+    page.save()
+
+    # query by ID
+    variables = {"id": graphene.Node.to_global_id("Page", page.id)}
+    response = staff_api_client.post_graphql(
+        PAGE_QUERY,
+        variables,
+        permissions=[permission_manage_pages],
+        check_no_permissions=False,
+    )
+    content = get_graphql_content(response)
+    assert content["data"]["page"] is not None
```

Comment:
IMO we should check if the return data is the page that we expected.

Reply:
@korycins

Reply:
Looks like a good suggestion. I just copy and paste the test and modify them to check the scenario. But for changes to master, we can extend them as Iga propsed.